### DISCUSSION TITLE
Morphooptiini ori - tessellation orientation import problem

### DISCUSSION THREAD

**Discussion by p-antonioni** [neperfepx/neper#841] [Q&A] [answered]
Hi,
I am trying to import a tessellation since i need to create a lamellar microstructure with lamellae oriented in a specific direction of the parent crystal.
I generated a tessellation with the following command:
`neper -T -n 30 -dim 2 -domain 'square(1,1)' -reg 1 -morpho 'gg' -ori 'Cube:normal(theta=5)' -transform 'rotate(1,0,0,4)' -o parent`
Then I tried to verify if I was importing it correctly  with the following command but i do not get the same orientation of the original tessellation:
`neper -T -n 30 -dim 2 -domain 'square(1,1)' -reg 1 -morpho 'gg' -morphooptiini 'file(parent.tess)' -o imported`
I also tried `-morphooptiini 'ori:file()'`, as a file I tried both the tesselation file and a file containing the orientations. 
Orientations are expressed in default way (rodrigues).
I have neper 4.8.2
Do you know what could be the issue?

Thanks in advance,
Paolo

**Comment by rquey** [comment, marked as answer]
With `-morphooptiini ori`, you also have to use `-ori from_morpho`.  In the doc, this appears under [-ori](https://neper.info/doc/neper_t.html#cmdoption-ori), but not under [-morphooptiini](https://neper.info/doc/neper_t.html#cmdoption-morphooptiini).  I'm going to fix this.

**Reply by p-antonioni** [reply]
Yes, missed that. Thanks for your reply and support!
